diff --git a/node_modules/@remix-run/dev/config.js b/node_modules/@remix-run/dev/config.js
index 06daccb..75652bc 100644
--- a/node_modules/@remix-run/dev/config.js
+++ b/node_modules/@remix-run/dev/config.js
@@ -58,11 +58,12 @@ async function readConfig(remixRoot, serverMode = serverModes.ServerMode.Product
   }
 
   let rootDirectory = path__namespace.resolve(remixRoot);
-  let configFile = path__namespace.resolve(rootDirectory, "remix.config.js");
+  let configFile = findConfig(rootDirectory, "remix.config");
   let appConfig;
 
   try {
-    appConfig = require(configFile);
+    appConfig = await import(configFile);
+    appConfig = appConfig?.default || appConfig;
   } catch (error) {
     throw new Error(`Error loading Remix config in ${configFile}\n${String(error)}`);
   }
@@ -218,4 +219,15 @@ function findEntry(dir, basename) {
   return undefined;
 }
 
+const configExts = [".js", ".mjs", ".cjs"];
+
+function findConfig(dir, basename) {
+  for (let ext of configExts) {
+    let file = path__namespace.resolve(dir, basename + ext);
+    if (fse__namespace.existsSync(file)) return path__namespace.resolve(dir, file);
+  }
+
+  return undefined;
+}
+
 exports.readConfig = readConfig;
